### ISSUE TITLE
showbio=false not working

### ISSUE THREAD

**Issue by tonyhensler** [smashballoon/instagram-feed#30]
showbio=false settings are not removing the bio from the header.
Works when unchecking the Remove Bio checkbox in the Customize tab.

## SITE/SERVER INFO: ##
WordPress Version:        4.9.4
PHP Version:              5.6.31-4+wpengine10
Web Server Info:          Apache

## ACTIVE PLUGINS: ##
Advanced Custom Fields PRO: 5.6.9
All In One SEO Pack: 2.4.4.1
Bloom: 1.2.23
Facebook Pixel: 1.6.0
Hustle: 6.0
Instagram Feed: 1.6.2
Pixel Cat Lite: 2.3.1
Redirection: 3.2
TablePress: 1.9
WP Google Maps: 6.4.09
WP Migrate DB: 1.0.2

**Comment by craigschlegel** [comment]
Hey tonyhensler,

Thanks for taking the time to report a potential issue! I'm having trouble recreating this one. I just tested using the shortcode setting and it worked fine in the latest version.

I noticed you're using WP Engine. They have an aggressive caching system that sometimes won't allow our feeds to update for awhile. Is it possible that the caching system was causing this?

Let me know if you have more thoughts!
